CodeEdit/Features/NavigatorSidebar/NavigatorSidebarToolbarTop.swift

```diff
@@ -19,11 +19,11 @@ struct NavigatorSidebarToolbarTop: View {
         SidebarDockIcon(imageName: "folder", title: "Project", id: 0),
         SidebarDockIcon(imageName: "vault", title: "Version Control", id: 1),
         SidebarDockIcon(imageName: "magnifyingglass", title: "Search", id: 2),
-        SidebarDockIcon(imageName: "shippingbox", title: "...", id: 3, disabled: true),
-        SidebarDockIcon(imageName: "play", title: "...", id: 4, disabled: true),
-        SidebarDockIcon(imageName: "exclamationmark.triangle", title: "...", id: 5, disabled: true),
+        SidebarDockIcon(imageName: "shippingbox", title: "Packages", id: 3, disabled: true),
+        SidebarDockIcon(imageName: "play", title: "Run", id: 4, disabled: true),
+        SidebarDockIcon(imageName: "exclamationmark.triangle", title: "Bugs", id: 5, disabled: true),
         SidebarDockIcon(imageName: "curlybraces.square", title: "...", id: 6, disabled: true),
-        SidebarDockIcon(imageName: "puzzlepiece.extension", title: "...", id: 7, disabled: true),
+        SidebarDockIcon(imageName: "puzzlepiece.extension", title: "Extensions", id: 7, disabled: true),
```

Comment:
This isn't really related to the git integration, isn't it?

Reply:
You're right, should I revert the change or keep it like that? I'm also implementing a run button and I forgot to revert it back.

Reply:
What do you think, @lukepistrol?